src/main/java/frc/robot/subsystems/intake/IntakeLoggedInputs.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.subsystems.intake;
+
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+public class IntakeLoggedInputs implements LoggableInputs {
+    double supMAX;
+    double infMAX;
+    boolean solenoid;
```

Comment:
These names aren't clear. The first two sound like they are the motor controllers themselves.

Reply:
Already changed